nikola/conf.py.in

```diff
@@ -562,6 +562,29 @@ GITHUB_COMMIT_SOURCE = True
 #    ".jpg": ["jpegoptim --strip-all -m75 -v %s"],
 # }
 
+# The executable for the filter "yui_compressor" from nikola.filters can
```

Comment:
Since `YUI_COMPRESSOR` is simply inserted in the command line, it can also be `java -jar ~/yui-compressor.jar`.

About the naming: I'll adjust that.